scipy/fft/_fftlog.py

```diff
@@ -242,37 +247,39 @@ def ifht(A, dln, mu, offset=0.0, bias=0.0):
     :math:`\mu` may be any real number, positive or negative.
 
     See `fht` for further details.
+    """
+    xp = array_namespace(A)
 
-    '''
+    if not from_uarray and is_numpy(xp):
+        return copy(_fftlog_uarray.ifht(A, dln, mu, offset=offset, bias=bias,
```

Comment:
Since this is a numpy array already, the `copy` doesn't look necessary

Reply:
Ah, I don't think this is needed anymore (or below). I remember adding it when `torch` was complaining about trying to `xp.asarray` an `np` array, but that won't be the case anymore.